source/documentation/deploying_services/index.md

```diff
@@ -1028,84 +1028,22 @@ You must:
     cf install-plugin conduit
     ```
 
-1. Start Conduit by running:
+1. Run the following command to connect to your service instance with the Redis client:
 
     ```
-    $ cf conduit SERVICE_NAME
+    $ cf conduit SERVICE_NAME -- redis-cli
     ```
 
     where `SERVICE_NAME` is a unique descriptive name for this service instance.
 
-    You will see the following output:
-
-    ```
-    service: SERVICE_NAME (redis)
-    host: 127.0.0.1
-    port: 7081
-    username:
-    password: REDIS_PASSWORD
-    db: DATABASE
-    ```
-
-    where `REDIS_PASSWORD` is the password given to you by Conduit.
-
-#### Install and configure stunnel
-
-1. Run the following command to install stunnel:
-
-    ```
-    brew install stunnel
-    ```
-
-    If you're not on a Mac, or don't have Homebrew, see the [stunnel
-    website](https://www.stunnel.org/downloads.html) [external link] for other
-    installation options.
-
-1. Create a stunnel configuration file that contains the following text:
-
-    ```ini
-    debug=7
-    options=NO_SSLv2
-    options=NO_SSLv3
-    foreground=yes
-    pid=/tmp/stunnel.pid
-    [redis]
-    client = yes
-    accept = 127.0.0.1:6379
-    connect = 127.0.0.1:7081
-    ```
-
-    where:
-    - `127.0.0.1:6379` is the default Redis host and port that stunnel will open (you will need to use a different port if you are already running a local Redis service instance)
-    - `127.0.0.1:7081` is the default Conduit host and port outputted by the `cf conduit SERVICE_NAME` command
-
-#### Connect your local machine to your Redis service instance
-
-1. Start stunnel by running:
-
-    ```
-    stunnel CONFIGURATION_FILE
-    ```
-
-1. Connect to your Redis service instance by running:
-
-    ```
-    redis-cli -a REDIS_PASSWORD
-    ```
-
-    where `REDIS_PASSWORD` is the password outputted by the `cf conduit SERVICE_NAME` command.
-
-You have now connected your local machine to your Redis service instance using Conduit and stunnel. You can test this connection with the Redis [PING](https://redis.io/commands/ping) command:
+You have now connected your local machine to your Redis service instance using Conduit. You can test this connection with the Redis [PING](https://redis.io/commands/ping) command:
 
 ```
 127.0.0.1:6379> PING
```

Comment:
@samcrang I don't have a running redis instance, could you paste the exact response for a PING please? Although I suspect it will be:
```
127.0.0.1:6379> PONG
```
:)

Reply:
```
$ cf conduit redis-test -- redis-cli
OK Connecting client
OK Targeting org govuk-paas
OK Targeting space tools
OK Deploying __conduit_52686__
OK Uploading __conduit_52686__ bits
OK Starting __conduit_52686__
OK Waiting for conduit app to become available
OK Fetching service infomation
OK Binding redis-test
OK Fetching environment
OK Starting port forwarding
OK Waiting for port forwarding
OK Preparing command: redis-cli
127.0.0.1:7081> PING
PONG
```

Reply:
Added